.circleci/config.yml

```diff
@@ -224,9 +224,9 @@ jobs:
         type: integer
         default: 1
       target:
-        description: makefile target
+        description: makefile target without test/e2e prefix
         type: string
-        default: "test/e2e"
+        default: ""
```

Comment:
should the default be `legacy` or `mix` ? Feels more understandable than ""

Reply:
Not sure, given that it's the `e2e` job I would guess `test/e2e` to be the default target. For a string, `""` is an ok default value. That is, if I don't specify a target at all, I'd expect _some_ default to be run.

Reply:
I think we should rename `test/e2e`, I'll do this in another PR